CONTRIBUTORS.md

```diff
@@ -0,0 +1,15 @@
+# Contributors List
+
+This is the list of Ed-Fi-ODS-Implementation's significant contributors.
+
+This does not necessarily list everyone who has contributed code, especially
+since many employees of one corporation may be contributing. To see the full
+list of contributors, see the revision history in source control or the [GitHub
+contributors
+graph](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Standard/graphs/contributors).
```

Comment:
Please update this link to make it redirect to the current repository

Reply:
Fixed